libraries/net/node.cpp

```diff
@@ -2666,6 +2661,11 @@ namespace graphene { namespace net { namespace detail {
             return;
           }
 
+          // append the remaining items to the peer's list
+          boost::push_back(originating_peer->ids_of_items_to_get, item_hashes_received);
+
+          originating_peer->number_of_unfetched_item_ids = blockchain_item_ids_inventory_message_received.total_remaining_item_count;
```

Comment:
Please remove this line, it's a duplicate of line 2563 (otherwise this would be a breaking change).
Same in BitShares.